include/proxy/proxy.h

```diff
@@ -1521,8 +1524,9 @@ struct explicit_conversion_adapter {
   T&& value_;
 };
 
-constexpr std::size_t invalid_size = std::numeric_limits<std::size_t>::max();
-constexpr constraint_level invalid_cl = static_cast<constraint_level>(
+inline constexpr std::size_t invalid_size =
```

Comment:
Is `inline` necessary?

Reply:
Yes. It makes the variable have external linkage